source/includes/atlas-sp.rst

```diff
@@ -0,0 +1,9 @@
+.. note::
+
+   Atlas Stream Processing provides MongoDB-native tooling to
+   continuously process streaming data, validate schemas, and
+   materialize views into either Atlas database collections or Apache
+   Kafka topics.
+
+   To learn more about Atlas Stream Processing, see the
```

Comment:
use substitutions

Also, use this formula:
To learn more, see |service| `Stream Processing <https://www.mongodb.com/docs/atlas/atlas-sp/overview/>`__.